### ISSUE TITLE
Loss is decreasing but accuracy stuck on 47% 

### ISSUE THREAD

**Issue by irum** [GPflow/GPflow#1491]
I have asked this question on stack overflow[ link ](https://stackoverflow.com/questions/62110184/accuracy-is-not-increasing-though-loss-is-decreasing) .
I suspect that there may be some error in prediction functions as it’s very weird that my loss Is decreasing but accuracy stuck on 47% after 10 epochs, then remain on that after few more epochs
But after 30 epochs, loss is still decreasing, however accuracy is now beginning to decrease rather than increase. 
This issue may relate to performance as I can not see any coding error in the training loop.

**Comment by awav** [comment]
I don't think we can classify this issue as an error. Unfortunately, there are too many unknowns, including lack of MFE. StackOverflow is the right place for discussing this matter.
